src/View/SSTemplateParser.peg

```diff
@@ -288,7 +288,7 @@ class SSTemplateParser extends Parser implements TemplateParser
             $arguments = $sub['Call']['CallArguments']['php'];
             $res['php'] .= "->$method('$property', [$arguments], true)";
         } else {
-            $res['php'] .= "->$method('$property', null, true)";
+            $res['php'] .= "->$method('$property', [], true)";
```

Comment:
This was needed to respect the typehint for `obj()` and `XML_val()` in `ViewableData`.